shotover/src/runner.rs

```diff
@@ -47,17 +47,11 @@ struct ConfigOpts {
     #[arg(long, value_enum, default_value = "human")]
     pub log_format: LogFormat,
 
-    /// Enable hot reloading functionality
+    /// Path for Unix socket used in hot reload communication.
+    /// If the socket exists, shotover will connect to it for hot reload.
+    /// If the socket doesn't exist, shotover will create it and enable hot reload server.
```

Comment:
The documentation should also include that the hot reload server will also be started (with retry on socket availability) when the the socket doesnt exist at first.